manageiq-providers-hawkular.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,lib}/**/*"]
 
-  s.add_runtime_dependency "hawkular-client", "~> 3.0.2"
+  s.add_runtime_dependency "hawkular-client", "~> 4.0.0"
```

Comment:
adding @moolitayer  - is this dependency on hawkular client still valid on k8s/ose providers considering the move to P8s?